fecgov/fecfile-web-app
# Add JavaScript lint check to CircleCI

Issue #126 
Add JavaScript lint check to CircleCI.

**How to test**
You can check out this branch, make a copy, make a linting error, and commit/push to see the failing build.
Example: https://app.circleci.com/pipelines/github/fecgov/fecfile-web-app/254/workflows/de2a3b34-8bd3-46c0-9bb1-a6c0949bc07b (I fixed the casing with [this commit](https://github.com/fecgov/fecfile-web-app/pull/193/commits/2931d17b8aa1f7a5c258765de3a88c922dda4ffe))

<img width="724" alt="Screen Shot 2022-03-31 at 4 01 53 PM" src="https://user-images.githubusercontent.com/31420082/161139799-288c4fbe-c786-4496-8ca6-984dd647c0cd.png">
